### DISCUSSION TITLE
Questions

### DISCUSSION THREAD

**Discussion by Teeraka** [lxqt/lxqt#1908] [Q&A]
Hi there, I don't know where to ask my questions so I hope you can help me.

I made a fresh lxqt install over centos 7 

But I am facing two problems, the first one is LXQT does not save my display preference even if I saved it.

The second one is I can not change my background wallpaper, actually there is nothing just dark screen.

I hope you can help me, I really love LXQT on my old computer :)

Thanks

Regards

**Comment by tsujan** [comment]
> the first one is LXQT does not save my display preference even if I saved it.

What do you mean by "display preference"? What do you mean by not being able to change your background wallpaper? Which version of LXQt?

If you want help, you need to be much more specific than this.
